.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -27,6 +27,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: ilammy/msvc-dev-cmd@v1.5.0
+      - uses: actions/setup-node@v1
```

Comment:
looks like v2 is the latest version https://github.com/marketplace/actions/setup-node-js-environment, or was there a problem with v2?